OptimizelySDK.Tests/UtilsTests/EventTagUtilsTest.cs

```diff
@@ -61,35 +54,45 @@ public void TestGetRevenueValue()
             };
 
             // Invalid data.
-            Assert.Null(EventTagUtils.GetRevenueValue(null));
-            Assert.Null(EventTagUtils.GetRevenueValue(invalidTag));
-            Assert.Null(EventTagUtils.GetRevenueValue(nullValue));
-            Assert.Null(EventTagUtils.GetRevenueValue(invalidValue));
-            Assert.Null(EventTagUtils.GetRevenueValue(invalidTagNonRevenue));
+            Assert.Null(EventTagUtils.GetRevenueValue(null, Logger));
```

Comment:
Please what was the sequence, don't change it. 
It's easy to read & review code.